pkgs/development/compilers/rust/1_30.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib
+, buildPackages
+, newScope, callPackage
+, CoreFoundation, Security
+, llvmPackages_5
+, llvm_7
+, pkgsBuildTarget, pkgsBuildBuild
+} @ args:
+
+import ./default.nix {
+  rustcVersion = "1.30.1";
+  rustcSha256 = "0aavdc1lqv0cjzbqwl5n59yd0bqdlhn0zas61ljf38yrvc18k8rn";
+
+  llvm = llvm_7;
```

Comment:
Any reason the newer rustc takes an older llvm?

Reply:
Oops I lost some changes when moving code around. Rust <1.34 needs LLVM 7.